src/cityreader/cityreader.py

```diff
@@ -17,11 +26,16 @@
 cities = []
 
 def cityreader(cities=[]):
-  # TODO Implement the functionality to read from the 'cities.csv' file
+  # Implement the functionality to read from the 'cities.csv' file
   # For each city record, create a new City instance and add it to the 
   # `cities` list
-    
-    return cities
+  with open('/Users/clintkunz/Python/Sprint-Challenge--Intro-Python/src/cityreader/cities.csv') as csvfile:
+    reader = csv.reader(csvfile)
+    for row in reader:
+      if row[0] != 'city':
```

Comment:
You can also do `next(reader)` above your for loop to skip the first line. https://www.programiz.com/python-programming/methods/built-in/next